definitions/npm/axios_v0.13.x/flow_v0.25.x-/axios_v0.13.x.js

```diff
@@ -1,22 +1,28 @@
-declare module 'axios' {
+declare module "axios" {
   declare interface AxiosXHRConfigBase<T> {
     adapter?: <T>(config: AxiosXHRConfig<T>) => Promise<AxiosXHR<T>>;
     auth?: {
-        username: string,
-        password: string
+      username: string,
+      password: string
     };
-    baseURL?: string,
+    baseURL?: string;
     progress?: (progressEvent: Event) => void | mixed;
     maxContentLength?: number;
-    maxRedirects?: 5,
+    maxRedirects?: number;
     headers?: Object;
     params?: Object;
     paramsSerializer?: (params: Object) => string;
-    responseType?: 'arraybuffer' | 'blob' | 'document' | 'json' | 'text' | 'stream';
+    responseType?:
+      | "arraybuffer"
```

Comment:
Looks like prettier did its thing here. Change line this back, comment out the `precommit` script in package.json, and push that commit up. I'll make sure it gets merged in.